index.html

```diff
@@ -33,16 +33,185 @@ <h1 class="name">Favour <br />Ezeugwa</h1>
+            <li>HTML/CSS</li>
+            <li>Ruby on Rails</li>
+            <li>Javascript</li>
+          </ul>
+          <button type="button" class="btn button-with-arrow">
+            See this project
+            <img src="./images/Union.png" alt="a forward icon" />
+          </button>
+        </div>
+      </div>
+      <div class="project-container">
+        <img
+          src="./images/Rectangle_6.png.png"
+          class="project-image"
+          alt="laptop"
+        />
+        <div class="project-info">
+          <p class="project-info-name">Project name goes here</p>
+          <ul class="project-languages">
+            <li>HTML/CSS</li>
+            <li>Ruby on Rails</li>
+            <li>Javascript</li>
+          </ul>
+          <button type="button" class="btn button-with-arrow">
+            See this project
+            <img src="./images/Union.png" alt="a forward icon" />
+          </button>
+        </div>
+      </div>
+    </section>
+    <section id="about-container">
+      <h1 class="about-heading">About</h1>
+      <p class="content">
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+        tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
+        veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea
+        eiusmod
+      </p>
+      <button type="button" class="btn download-button">
+        Get my cv
+        <img src="./images/ic_download.png" alt="a download button" />
+      </button>
+      <div class="section-about">
+        <!-- Blue about section -->
+        <div class="languages blue">
+          <h1>languages</h1>
+          <ul class="container-list">
+            <li><span>&bull; JavaScript(ES5/ES6)</span></li>
+            <li><span>&bull; Ruby</span></li>
+            <li><span>&bull; HTML</span></li>
+            <li><span>&bull; Css</span></li>
+          </ul>
+          <img class="about-img" src="./images/h.png" alt="hash tag" />
+        </div>
+        <!-- Red about section -->
+        <div class="languages red">
+          <h1>Frameworks</h1>
+          <ul class="container-list">
+            <li><span>&bull; Bootstrap</span></li>
+            <li><span>&bull; Ruby on Rails</span></li>
+            <li><span>&bull; RSpec</span></li>
+            <li><span>&bull; Capibara</span></li>
+            <li><span>&bull; Selenuim</span></li>
+          </ul>
+          <img
+            class="about-img"
+            src="./images/ic_framework.png"
+            alt="framework icon"
+          />
+        </div>
+        <!-- Orange about section -->
+        <div class="languages orange">
+          <h1>Skills</h1>
+          <ul class="container-list">
+            <li><span>&bull; Codekit</span></li>
+            <li><span>&bull; Codepen</span></li>
+            <li><span>&bull; Github</span></li>
+            <li><span>&bull; GitLab</span></li>
+            <li><span>&bull; Terminal</span></li>
+          </ul>
```

Comment:
Kindly wrap the icon names around an a tag, since they would navigate to a webpage when they are clicked. 

## :slightly_smiling_face:

Reply:
Hello Uche,

Good morning. 

I will work on all suggestions made, however, I used template 5 in the Figma template provided for the curriculum. 
Can you help me with the steps on how to use the same Pull Request submitted for the first review?